### PR TITLE
Adds two new intermediate beaker types to chemistry

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
@WJohn the new sprites are just modded versions of the large one, the 120u is just slightly opaque white whereas the other one is a slighly golden taller one. 

We probably want 3/4 beakers at some point but I'm the wrong guy for that.

:cl: Naksu
tweak: Plastic is now a valid material for lathes
add: Added two new beaker types, 120u and 180u. The 120u beaker can be printed off the medical protolathe with just glass and plastic, the 180u requires some mining materials and tech.
/:cl:

### CODE REVIEW

File: code/modules/reagents/reagent_containers/glass.dm [hunk 1]
```diff
@@ -165,6 +165,29 @@
...
+	icon_state = "beakerwhite"
+
+/obj/item/reagent_containers/glass/beaker/meta
+	name = "metamaterial beaker"
+	desc = "A large beaker. Can hold up to 180 units."
+	icon_state = "beakergold"
+	materials = list(MAT_GLASS=2500, MAT_PLASTIC=3000, MAT_GOLD=1000, MAT_TITANIUM=1000)
+	volume = 180
+	amount_per_transfer_from_this = 10
+	possible_transfer_amounts = list(10,15,20,25,30,60,120)
```
Comment (XDTM) [inline code/modules/reagents/reagent_containers/glass.dm]: 180 too?

Comment (vuonojenmustaturska) [reply code/modules/reagents/reagent_containers/glass.dm]: I guess

**Comment by ExcessiveUseOfCobblestone** [comment]
> 180 ml beaker

send in the bug(gy) squad

**Comment by arsserpentarium** [comment]
Just replace 50 and 100 with 60 and 120.
Because it's not about additional volume, but about dividers.

**Comment by ExcessiveUseOfCobblestone** [comment]
why don't we just give chem buttons that are the chem straight w/o recipe and scale the energy so it always fits your beaker and you don't have carpal tunnel by having to type the recipe or ctrl+c ctrl+v?

Why don't we just give chem 100u of each chem roundstart that can be synthed so they don't have any carpal tunnel at all except picking up the beaker?

Why don't we...

**Comment by factoryman942** [comment]
just give chemists backpack roundstart hellfoam nades, nanites and romerol tbh

**Comment by Iatots** [comment]
Please no, glassware with arbitratry different tints gives me OCD.

**Comment by Buggy123** [comment]
>send in the bug(gy) squad

sup

**Pull request merged**